src/main/java/com/zaxxer/hikari/pool/PoolEntry.java

```diff
@@ -76,8 +77,21 @@ final class PoolEntry implements IConcurrentBagEntry
     */
    void recycle()
    {
+      // endRequest Call just to let driver know that the connection has been returned to pool.
```

Comment:
Do away with these comments as they aren't consistent with the existing codebase.